### ISSUE TITLE
Clair v4 support

### ISSUE THREAD

**Issue by mieliespoor** [Inedo/inedox-clair#2]
Looking at the API urls in `ClairVulnerabilitySource.cs` it would appear that this extension is supporting only version 2 of clair.

Is there any plans to upgrade it to support v4.3 of clair - current latest?

**Comment by whatatripp** [comment]
Yes, but there were some showstopping bugs in their API that made the move impossible earlier. It's on our roadmap to re-evaluate and investigate this year.

**Comment by mieliespoor** [comment]
Should this issue then not be kept open until the upgrade to 4 happens instead of just dismissing it?

**Comment by WojRep** [comment]
Any news ?
